### ISSUE TITLE
Algolia dark mode highlighted result has low contrast

### ISSUE THREAD

**Issue by theetrain** [withastro/docs#1324]
Hello, while browsing the site in dark mode I observed highlighted results in the Algolia search modal have white text on a bright orange background, which has very low contrast as you'll see below:

![image](https://user-images.githubusercontent.com/12798751/185021332-2cdb3020-2e41-4fcd-8fa9-44c9cfa813cf.png)

To meet WCAG AAA standards, an easy win would be choosing a near-black text colour for highlighted results.
I suggest:

- `#ff995f` text on a `#141415` background (note this orange is different from the `#ff7e33` orange used throughout the site)
- `#141415` text against the orange background

**Comment by Yan-Thomas** [comment]
Thanks for reporting this @theetrain. It's worth pointing out that [contrast checkers don't work so well with orange](https://www.bounteous.com/insights/2019/03/22/orange-you-accessible-mini-case-study-color-ratio/), so I'll do some research to be sure of what is the best approach here.

This looks like a vry good first issue! Would you like to open a PR fixing this?

**Comment by theetrain** [comment]
@Yan-Thomas thanks for the insight! Sure I can open a PR with my proposed colours and we can have a back-and-forth from there. I hope to get around to this within the next couple of days.

**Comment by delucis** [comment]
Thanks @theetrain! Let us know if you run into any issues.
